app/src/lib/stores/git-store.ts

```diff
@@ -148,6 +146,9 @@ export class GitStore extends BaseStore {
     private readonly statsStore: StatsStore
   ) {
     super()
+
+    const rawTags = localStorage.getItem(getTagsToPushKey(repository))
+    this._tagsToPush = rawTags ? JSON.parse(rawTags) : []
```

Comment:
I've opted to store the list of tags in JSON format since this will be a short list of strings. We could also have just joined the array of tags using whitespace since tags can't have whitespace, but I'm erring on the safe side here.